x/uibc/keys.go

```diff
@@ -0,0 +1,28 @@
+package uibc
+
+import "github.com/umee-network/umee/v4/util"
+
+const (
+	// ModuleName defines the module name
+	ModuleName = "uibc"
+
+	// StoreKey defines the primary module store key
+	StoreKey = ModuleName
+
+	// RouterKey is the message route for uibc
+	RouterKey = ModuleName
+)
+
+var (
+	KeyPrefixForIBCDenom = []byte{0x01}
```

Comment:
```suggestion
	KeyPrefixDenomQuota = []byte{0x01}
```

Reply:
Let's use same naming structure as in https://github.com/umee-network/umee/blob/main/x/oracle/types/keys.go